pom.xml

```diff
@@ -234,7 +234,7 @@
     <org.apache.maven.plugins.maven-war-plugin.version>3.3.1</org.apache.maven.plugins.maven-war-plugin.version>
     <org.codehaus.mojo.build-helper-maven-plugin.version>3.2.0</org.codehaus.mojo.build-helper-maven-plugin.version>
     <org.apache.maven.plugins.maven-failsafe-plugin.version>3.0.0-M7</org.apache.maven.plugins.maven-failsafe-plugin.version>
```

Comment:
- [ ] maven-failsafe-plugin is also 3.0.0-M5

Reply:
Fixed in this commit
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/1000/commits/6e70a7dec432491b7c61759edb8e0f62497fa713